test/functional/feature_csv_activation.py

```diff
@@ -9,8 +9,8 @@
 BIP 112 - CHECKSEQUENCEVERIFY
 BIP 113 - MedianTimePast semantics for nLockTime
 
-mine 82 blocks whose coinbases will be used to generate inputs for our tests
-mine 345 blocks and seed block chain with the 82 inputs will use for our tests at height 427
+mine 83 blocks whose coinbases will be used to generate inputs for our tests
+mine 344 blocks and seed blockchain with the 83 inputs used for our tests at height 427
```

Comment:
nit, I think there's no need to change "block chain"; see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words

Reply:
I reverted this change.